sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/parquet/ParquetIOIT.java

```diff
@@ -197,6 +198,9 @@ private void collectAndPublishMetrics(PipelineResult result) {
           return NamedTestResult.create(uuid, timestamp, "run_time", runTime);
         });
 
+    metricSuppliers.add(
```

Comment:
If `datasetSize` is not provided (this can happen, because the option is not required) this metric value is null. Is this intentional?